assets/dev/scss/frontend/widgets/accordion.scss

```diff
@@ -94,6 +94,7 @@
 
 	> .elementor-widget-accordion {
 		// Fix - Accordion widget disappears because the 'flex column' setting causes an undefined width inside the flex container (ED-7524).
-		width: var( --container-widget-width, 100% );
+		width: var( --container-widget-width );
+		flex-grow: var( --container-widget-flex-grow );
```

Comment:
Could we use `--flex-grow` instead of `flex-grow`, in order to give the users more freedom and more control?

Reply:
Done in all places